api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/MetricHandler.java

```diff
@@ -847,7 +75,7 @@ public void onFailure(Throwable t) {
     public void findMetrics(
             @Suspended final AsyncResponse asyncResponse,
             @PathParam("tenantId") final String tenantId,
-        @ApiParam(value = "Queried metric type", required = true, allowableValues = "[num, avail, log]")
+        @ApiParam(value = "Queried metric type", required = true, allowableValues = "[gauge, avail, log]")
```

Comment:
For consistency I think the `type` parameter should accept the values,

* availability
* gauges
* counters

Since we have not implemented anything for events/logs, I think `log` should be removed.